README.md

```diff
@@ -43,20 +43,23 @@ its GUI. You basically have two options.
 **Additional Notes**: [Some packages we
 love](https://github.com/TypeStrong/atom-typescript/blob/master/docs/packages.md).
 
-## Reviews
+## Reviews:
```

Comment:
This `:` is inconsistent with the rest of the headings. So I don't know why you chose to put it here.